tomviz/python/AutoCrossCorrelationTiltImageAlignment.py

```diff
@@ -66,6 +66,12 @@ def transform_scalars(self, dataset):
 
         utils.set_array(dataset, tiltSeries)
 
+        # Assign Negative Shifts when Shift > N/2.
+        max_indeces_X = np.where(offsets > tiltSeries.shape[0] / 2)
+        offsets[max_indeces_X] = offsets[max_indeces_X] - tiltSeries.shape[0]
```

Comment:
This still affects both X and Y when X is too high.  I just tested it by setting the X offset for the first slice to well above the size.  This subtracted `tiltSeries.shape[0]` from both the X and Y offsets of the first slice.

Your condition to find X offsets that are too high also found the Y offset I found that was "too high" (actually it was a valid Y offset, I just forced it to be higher than the X limit.

Offsets (according to my print statement) look like this (note this print is from before I started changing values):
```
[ [ 98.  43.]
 [ 98.  36.]
 [ 98.  30.]
 [ 98.  25.]
 [ 98.  21.]
 [ 98.  20.]
...
]
```

So your search needs to take into account that there are two columns and only search the X or Y column, not both.